django_addanother/views.py

```diff
@@ -9,6 +10,9 @@ class PopupMixin(object):
     """Mixin for :class:`~django.views.generic.edit.CreateView` classes that
     handles the case of the view being opened in an *add another* popup window.
     """
+
+    POPUP_ACTION = 'add'
```

Comment:
Factor this into three classes `BasePopupMixin`, `AddPopupMixin`, `ChangePopupMixin`